docs/Privacy.md

```diff
@@ -14,5 +14,8 @@ Currently telemetry is only implemented for Windows builds, but may be expanded
 #### Technical Details
 The Windows provider uses the [TraceLogging](https://docs.microsoft.com/en-us/windows/win32/tracelogging/trace-logging-about) API for its implementation.
 
-Windows ML and onnxruntime C APIs allow telemetry collection to be turned on/off (see API pages for details). In all Official builds, the Windows ML API enables telemetry collection.
+Windows ML and onnxruntime C APIs allow telemetry collection to be turned on/off (see API pages for details).
+In all Official builds:
+- Telemetry collection is *disabled* for calls to the ORT API.
+- Telemetry collection is *enabled* for calls to the Windows ML API.
```

Comment:
This looks weird:

![image](https://user-images.githubusercontent.com/5521887/73701178-8a296580-469d-11ea-8944-c25df104851e.png)

The C API link should follow `see API pages for details`, now it's in the end of the `In all Official builds` list

Reply:
moved around.

Reply:
moved around

Reply:
moved around.